tests/metrics/test_representation.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+
+import torch
+from torchmetrics import Metric
+from typing import List, Dict, Union
+import numpy as np
+from src.metrics.representation import RecallAtKMeter, PrecisionAtKMeter, MeanAveragePrecisionAtKMeter, NDCGAtKMeter
+from src.metrics.representation import DatasetType, MetricDistance
+
+
+vectors = {
```

Comment:
You can create `utils.py` and move every repeated code blocks there (DRY principle)